conductr_cli/validation.py

```diff
@@ -169,6 +169,22 @@ def handler(*args, **kwargs):
     return handler
 
 
+def handle_read_timeout_error(func):
+    def handler(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except ReadTimeout as err:
+            log = get_logger_for_func(func)
+            log.error('Timed out waiting for response from the server: {}'.format(err.args[0]))
```

Comment:
Do we not want to convey possible reasons why it timed out e.g. roles being unmatched?